### PR TITLE
fixed the issue related to the tooltip preivew on mouseEvent

### PR DESCRIPTION
**Pull request by KaleemNeslit**
## Link to Issue
Closes: #9651 

## Description of Changes
- Improved tooltip behaviour by conditionally setting anchorEl based on event type, ensuring responsive show/hide functionality.
- added the <Div>  wrapper in <CWButton /> onMouseEnter events was not triggering on that component .

## "How We Fixed It"

- Streamlined the handleInteraction function to toggle anchorEl using the e.type check .
- Adjusted <CWButton /> usage within the QuickTokenLaunchForm page by placing the interaction logic directly on the containing <div>, improving encapsulation and simplifying component interaction management

## Test Plan
  - goto the communities and go to lunch token and click the randomize more than 5 times you can see the tooltip behaviour tip will appear onMouseLeaveEvent . 
  

https://github.com/user-attachments/assets/e8872cde-b1f0-4277-9436-a3d4a1c9eb5b

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/pages/Communities/IdeaLaunchpad/QuickTokenLaunchForm/QuickTokenLaunchForm.tsx [hunk 1]
```diff
@@ -450,22 +450,25 @@ export const QuickTokenLaunchForm = ({
...
-                        type="button"
-                        disabled={
-                          isProcessingProfileImage ||
-                          isCreatingQuickToken ||
-                          isMaxTokenIdeaLimitReached
-                        }
-                        onClick={() => {
-                          generateIdea().catch(console.error);
-                        }}
+                      <div
```
Comment (mzparacha) [inline packages/commonwealth/client/scripts/views/pages/Communities/IdeaLaunchpad/QuickTokenLaunchForm/QuickTokenLaunchForm.tsx:453]: does it not work with button?

Comment (KaleemNeslit) [reply packages/commonwealth/client/scripts/views/pages/Communities/IdeaLaunchpad/QuickTokenLaunchForm/QuickTokenLaunchForm.tsx:453]: yes in button mouseEnter events is not triggering that why tooltip show in mouseLeave and then hide in mouseLeave.

Comment (masvelio) [reply packages/commonwealth/client/scripts/views/pages/Communities/IdeaLaunchpad/QuickTokenLaunchForm/QuickTokenLaunchForm.tsx:453]: it definitely should work in buttons. The one case that it does not is when the button is disabled, then the events are not getting through.

**Review by masvelio — APPROVED** [review]

File: packages/commonwealth/client/scripts/views/pages/Communities/IdeaLaunchpad/QuickTokenLaunchForm/QuickTokenLaunchForm.tsx (hunk 1 above)
Comment (KaleemNeslit) [reply packages/commonwealth/client/scripts/views/pages/Communities/IdeaLaunchpad/QuickTokenLaunchForm/QuickTokenLaunchForm.tsx:453]: yes button is disable after 5 try @masvelio

**Pull request merged**